### PR TITLE
Feature/XIVY-14000 add playwright test for detail view

### PR DESCRIPTION
**Pull request by weissreto**
Add test for variable details view

### CODE REVIEW

File: integrations/standalone/tests/mock/variable-editor-detail.spec.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  });
+
+  test('new password variable', async () => {
+    await editor.tree.row(0).click();
+    await editor.add.click();
+    await editor.tree.row(11).click();
+    const details = editor.details;
+    await details.expectTitle('Variables Editor - ');
+
+    await details.fill('myName', 'myValue', 'This is myName with a value of myValue', 'Password');
```
Comment (ivy-lgi) [inline integrations/standalone/tests/mock/variable-editor-detail.spec.ts:37]: After changing the metadata type to `Password`, maybe we should also check that the value input field now has the type `password`?

File: integrations/standalone/tests/mock/variable-editor-detail.spec.ts [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  });
+
+  test('new daytime variable', async () => {
+    await editor.tree.row(0).click();
+    await editor.add.click();
+    await editor.tree.row(11).click();
+    const details = editor.details;
+    await details.expectTitle('Variables Editor - ');
+
+    await details.fill('myName', '12:15', 'This is myName with a value of myValue', 'Daytime');
```
Comment (ivy-lgi) [inline integrations/standalone/tests/mock/variable-editor-detail.spec.ts:56]: After changing the metadata type to `Daytime`, maybe we should also check that the value input field now has the type `daytime`?

File: integrations/standalone/tests/mock/variable-editor-detail.spec.ts [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    await details.enumValue.choose('Wednesday');
+    await details.enumValue.expectValue('Wednesday');
+
+    await details.enumValue.choose('Monday');
+
+    await details.listOfPossibleValues.deleteValue('Tuesday');
+    await details.listOfPossibleValues.deleteValue('Wednesday');
+
+    await details.listOfPossibleValues.expectValues(['Monday']);
+  });
```
Comment (ivy-lgi) [inline integrations/standalone/tests/mock/variable-editor-detail.spec.ts:133]: There are some edge cases here we had to handle that might want to get tested?
- What If you delete the element currently selected as the value? -> I think the value becomes empty.
- What if you add the same value twice? -> I think it works but still only shows up once in the value selection.

**Review by ivy-lgi — APPROVED** [review]
Maybe some things to add, otherwise good :+1:
